standard.tex

```diff
@@ -26,67 +26,77 @@
+Here are the rules for all the design and assembly decisions. If you want to break one of these rules, make sure you have a very good excuse. If you think a rule is unclear, feel free to elaborate on it. If you think there's a good rule that should be added, please write it in the margin or pull request the repository (\url{https://github.com/waterloo-rocketry/electrical\_standard}).
 
-%start of connections subsection -- {{{
+%Connections -- {{{
 \subsection{Connections}
 \begin{itemize}
-\item Don't solder stranded wire to protoboard. Don't solder stranded wire to a connector. It gets brittle and it breaks. Use crimped connections, screw terminals, etc instead.
-\item If you're making a crimped connection, use a proper crimp tool if at all possible. Pliers work in a pinch, but the connection is messier and more likely to fail. Good crimp tools are expensive but worth it. (If you're in MME, you can borrow one from MESS.)
-\item If you absolutely need to solder stranded core to something, make sure it's very strain relieved (I personally like P clamps, but zip ties work too).
-\item Crimped connections should be done on stranded wire only. Don't crimp onto solid core. It's weak and it will break, often when you least want it to.
-\item All exposed wire should be heatshrunk; all heatshrink should be clear (IREC regulation).
-\item If you screw up a soldering job (we all do it), redo the job. When soldering surface mount components, inspect solder joints with a small magnifying glass.
+\item Crimped connections are preferable to soldered connections. Bad crimped connections are easier to identify than bad solder joints. Additionally, it is easier to fix a bad crimp in the field since it does not require powered tooling.
+\item If you're making a crimped connection, use a proper crimp tool if at all possible. Pliers are not an acceptable alternative (see ESRA safety critical wiring guidelines). We have proper crimp tools for: generic butt splice and ring/spade connectors, Dupont connectors, and RJ45 connectors.
+\item Use ferrules for stranded wire that goes into a screw terminal.
+\item Crimped connections should be done on stranded wire only.
+\item Wire should only be soldered to connectors that are specifically made for it (eg. connectors with solder cups). Stranded wire should be used in the vast majority of cases since solid core wire becomes brittle and prone to breakage (exception: wire soldered onto protoboard should be solid core).
+\item All solder joints should be strain relieved with zipties or P clamps.
+\item If you screw up a soldering job, redo the job. When soldering surface mount components, inspect solder joints with a small magnifying glass.
 \item All connections must be tug proof.
+\item All exposed wire should be heatshrunk; all heatshrink should be clear (IREC regulation).
```

Comment:
The amount of exposed wire, even if heatshrunk, should be minimized and insulation should go up as close to a connector as possible (based on feedback from one of the judges during the poster session).

Reply:
fixed.